concordium-node/src/configuration.rs

```diff
@@ -600,6 +600,12 @@ pub struct CommonConfig {
         env = "CONCORDIUM_NODE_NO_LOG_TIMESTAMP"
     )]
     pub no_log_timestamp: bool,
+    #[structopt(
+        long = "log-config",
+        help = "Configure logging with a log4rs configuration file",
```

Comment:
Please add that this disables logging to stdout/err.

I also don't think the name of the library we are using should be in the help text.

Reply:
The configuration file determines whether logging is to stdout, stderr, a file or some combination. If we don't specify "log4rs", there's no clue for the user to figure out the configuration format.

Reply:
I think a common way for this is to include a default file, or a link to it, with documentation in the file, in the form of comments, of what the options do. It is more user friendly than a name of a crate which has fairly poor user documentation. We also don't say what features the crate is used with, so this help line is not really enough.